demisto_sdk/commands/validate/tests/RM_validators_test.py

```diff
@@ -314,3 +317,54 @@ def test_IsReadmeExistsValidator_is_valid(
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+def test_IsContainDemistoWordValidator_is_valid():
+    """
+    Given
+    content_items.
+        - Case 1: Two valid pack_metadatas:
+            - 1 pack with valid readme text.
+            - 1 pack with an empty readme.
+        - Case 2: One invalid pack_metadata with a readme that contains the word 'demisto'.
```

Comment:
No need to put this line because it is related to the other test. (same for the 'Then' part).